packages/at_client/lib/src/client/remote_secondary.dart

```diff
@@ -211,13 +199,6 @@ class RemoteSecondary implements Secondary {
     return false;
   }
 
-  Future<void> _restartCallBack(
-      String command, Function notificationCallBack, String privateKey) async {
-    logger.info('auto restarting monitor');
-    // ignore: deprecated_member_use_from_same_package
-    await monitor(command, notificationCallBack, privateKey);
-  }
-
```

Comment:
Removed the above stuff because it is not part of the Secondary, and was being used only by code which has now been removed from AtClientImpl